azure-pipelines/compare-refdata.yml

```diff
@@ -38,26 +34,18 @@ jobs:
     steps:
       - template: templates/default.yml
         parameters:
-          fetchRefdata: false  # See the comment below.
+          fetchRefdata: true
+          refdataRepo: 'github'
```

Comment:
`compare-refdata` pipeline is simplified by using GitHub source instead of Azure (thanks to our new sponsor GitHub we have free LFS).